lib/src/controls/better_player_material_controls.dart

```diff
@@ -507,6 +511,26 @@ class _BetterPlayerMaterialControlsState
     );
   }
 
+  Widget _buildVideoTitle() {
+    final videoTitle =
+        _betterPlayerController?.betterPlayerConfiguration.videoTitle ?? '';
+
+    return AnimatedOpacity(
+      opacity: controlsNotVisible ? 0.0 : 1.0,
+      duration: _controlsConfiguration.controlsHideTime,
+      child: Padding(
+        padding: EdgeInsets.only(
+          right: 16,
+          left: !_controlsConfiguration.enablePlayPause ? 16 : 0,
+        ),
+        child: Text(
```

Comment:
poka jak to. w orientacji pionowej wyglada

Reply:
![Uploading Simulator Screenshot - iPad (10th generation) - 2024-06-12 at 09.04.41.png…]()